tests/integration-test/src/tests/utils.rs

```diff
@@ -0,0 +1,98 @@
+use std::{process::Command, thread::sleep, time::Duration};
+
+use anyhow::Result;
+use log::debug;
+
+const DEFAULT_BPFD_IFACE: &str = "eth0";
+const XDP_PASS_IMAGE: &str = "quay.io/bpfd/bytecode:xdp_pass";
+
+/// Exit on panic as well as the passing of a test
+pub struct ChildGuard {
+    name: &'static str,
+    child: std::process::Child,
+}
+
+impl Drop for ChildGuard {
+    fn drop(&mut self) {
+        if let Err(e) = self.child.kill() {
+            println!("Could not kill {}: {}", self.name, e);
+        }
+        if let Err(e) = self.child.wait() {
+            println!("Could not wait for {}: {}", self.name, e);
+        }
+    }
+}
+
+/// Spawn a bpfd process
+pub fn start_bpfd() -> Result<ChildGuard> {
+    debug!("Starting bpfd");
+    let bpfd_process = Command::new("bpfd").spawn().map(|c| ChildGuard {
+        name: "bpfd",
+        child: c,
+    })?;
+    sleep(Duration::from_secs(2));
+    debug!("Successfully Started bpfd");
+    Ok(bpfd_process)
+}
+
+/// Check for a specified bpfd interface environmental, otherwise set a default
+pub fn read_iface_env() -> &'static str {
+    option_env!("BPFD_IFACE").unwrap_or(DEFAULT_BPFD_IFACE)
+}
+
+/// Install an xdp_pass program with bpfctl
+pub fn add_xdp_pass(iface: &str, priority: &str) -> String {
+    let output = Command::new("bpfctl")
+        .args([
+            "load",
+            "--iface",
+            iface,
+            "--priority",
+            priority,
+            "--from-image",
+            XDP_PASS_IMAGE,
+        ])
+        .output()
+        .expect("failed to install bpfd xdp_pass program");
+
+    let stdout = String::from_utf8(output.stdout).unwrap();
+    assert!(!stdout.is_empty());
+    debug!("Successfully added xdp_pass program: {:?}", stdout.trim());
+
+    stdout
+}
+
+/// Delete a bpfd program using bpfctl
+pub fn bpfd_del_program(bpfd_iface: &str, uuid: &str) {
+    Command::new("bpfctl")
```

Comment:
You should use `Command::cargo_bin` here since it will pick it up from the workspace - from `assert_cmd`
i.e it will look in`${workspace_root}/target/debug/`